stream-projects/do-something/routes/admin/do-something/create/page.tsx

```diff
@@ -0,0 +1,19 @@
+import CreateCollectibleForm from "../../../../components/create-collectible-form/create-collectible-form.tsx";
+import { enableLegendStateReact, React, toDist, useStyleSheet } from "../../../../deps.ts";
+import styleSheet from "./page.scss.js";
+
+enableLegendStateReact();
+
+function CreateCollectiblePage() {
+  useStyleSheet(styleSheet);
```

Comment:
We'll want to add a check that the authenticated user has permissions to create collectibles. Should be able to yoink the same pattern we use for mod tools in mogul-menu: https://github.com/trufflehq/mogul-menu/blob/8165767328dd94f705c069d83a1d0bf96de8f547/components/activities/activities-tab/activities-tab.tsx#L72

Reply:
actually looks like we're not doing any upsert here, it's just for copying over to the old admin. disregard